docs/_static/css/dark.css

```diff
@@ -1,6 +1,5 @@
 /* links */
-
-a,
+.rst-content a,
 a:visited {
     color: #aaddff;
 }
```

Comment:
This made links in page header dark blue:

![contract-metadata-header-dark-mode](https://user-images.githubusercontent.com/137030/133289716-c438cf85-4656-4a0c-8078-368844e3d6f3.png)